--- kxmleditor-1.1.3/kxmleditor/kxmleditor.desktop.orig	2004-07-11 18:16:58.000000000 +0200
+++ kxmleditor-1.1.3/kxmleditor/kxmleditor.desktop	2004-09-14 14:24:53.049404512 +0200
@@ -4,9 +4,10 @@
 Icon=kxmleditor.png
 DocPath=kxmleditor/index.html
 Comment=XML editor
+Comment[pl]=Edytor XML
 Comment[fi]=XML-editori
 Terminal=false
-Name=KXML Editor
+Name=Kxmleditor
 MimeType=text/xml;
 Encoding=UTF-8
-Categories=Application;Development;TextEditor;KDE;
+Categories=Qt;KDE;Development;TextEditor;
